### DISCUSSION TITLE
Respuesta de una pregunta del directo de hoy.

### DISCUSSION THREAD

**Discussion by maxkario** [Launch-X-Latam/MisionFrontEnd#267] [General]
Alguien en el direto hizo esta pregunta. Esta duda la tuve hace unos años y cuando comence a aprender HTML me la resolvió este video que ví de EDTeam.

![Clipboard02](https://user-images.githubusercontent.com/59313980/198171591-6aeb7b79-a77f-476b-89fa-f80176108741.jpg)

https://youtu.be/oyjnPmJkN8w

Espero que les sirva la aclaración. Saludos.

**Comment by doguedogue** [comment]
gracias 🚀
